src/mds/MDCache.cc

```diff
@@ -6708,7 +6708,7 @@ std::pair<bool, uint64_t> MDCache::trim(uint64_t count)
 	  continue;
 	list<CDir*> ls;
```

Comment:
@Vicente-Cheng do we need lines 6709 and 6710? I see this removed in the parent commit.

Reply:
@ajarr, I think we still need 6709 and 6710.

That two lines are removed with the following commits:
1. 1b1714c50b mds: convert get_*dirfrags to use vector
  - this commit changes the method about get_*dirfrags so we don't need to declare ls here.
2. 5eea6761cc mds: trim pinned and empty subtrees
  - this commit changes some conditions in MDCache::trim so the `continue` would be removed with the new condition.

unless we introduce the above commits, I think we still need two lines.

thanks!!

Reply:
> @ajarr, I think we still need 6709 and 6710.
> 
> That two lines are removed with the following commits:
> 
> 1. [1b1714c](https://github.com/ceph/ceph/commit/1b1714c50b4c97cb1906407490f828d4ba4cfc41) mds: convert get_*dirfrags to use vector
> 
> * this commit changes the method about get_*dirfrags so we don't need to declare ls here.
> 
> 1. [5eea676](https://github.com/ceph/ceph/commit/5eea6761cc7e5950810cb0a625d49bd3f788183c) mds: trim pinned and empty subtrees
> 
> * this commit changes some conditions in MDCache::trim so the `continue` would be removed with the new condition.
> 
> unless we introduce the above commits, I think we still need two lines.

@Vicente-Cheng the 2 commits that you mention are part of PRs that won't make it to nautilus. I suggest removing lines 6709 and 6710 as done in the original commit.

@batrick what do you think?

> 
> thanks!!

Reply:
@ajarr, ahh..
Yes, you are right, I focus the wrong line.
6709 and 6710 should be removed.
thanks for helping check.

Reply:
updated! Please help to check again.